src/components/data-management/UploadDetailsModal.jsx

```diff
@@ -29,12 +28,14 @@ const UploadDetailsModal = (props) => {
     if (replacementFileBundle) {
       // we'll need to remove the hard-coded 10x tech type once we start
       // supporting other types and save the chosen tech type in redux
```

Comment:
in order to make it more visible that this is hardcoded and it is hardcoded here, why don't you create a const "TECH_TYPE", make it equal to `10X Chromium` and put it at the top of this file?